src/client/app/components/snippet.info.ts

```diff
@@ -16,6 +17,11 @@ import { Strings } from '../helpers';
                     <label class="ms-Label">${Strings.descriptionLabel}</label>
                     <textarea class="ms-TextField-field ms-font-m" [(ngModel)]="snippet.description" placeholder="Description of the snippet"></textarea>
                 </div>
+
+                <div *ngIf="url" class="ms-TextField ms-TextField--multiline">
+                    <label class="ms-Label">${Strings.urlLabel}</label>
+                    <textarea class="ms-TextField-field ms-font-m" [(ngModel)]="url" disabled="true"></textarea>
```

Comment:
For the gist URL in the about snippet dialog, should we just do "Gist ID:  Open in browser" with a hyperlink?